modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -26,6 +26,31 @@
 class Link_In_Bio extends Widget_Base {
 	const TAB_ADVANCED = 'advanced-tab-custom';
 
+	protected static $platform_icon_mapping = [
+		'Facebook'    => 'fab fa-facebook',
+		'Instagram'   => 'fab fa-instagram',
+		'LinkedIn'    => 'fab fa-linkedin-in',
+		'Pinterest'   => 'fab fa-pinterest',
+		'TikTok'      => 'fab fa-tiktok',
+		'X (Twitter)' => 'fab fa-x-twitter',
+		'YouTube'     => 'fab fa-youtube',
+		'Apple Music' => 'fab fa-apple',
+		'Behance'     => 'fab fa-behance',
+		'Dribble'     => 'fab fa-dribbble',
+		'Spotify'     => 'fab fa-spotify',
+		'SoundCloud'  => 'fab fa-soundcloud',
+		'Vimeo'       => 'fab fa-vimeo-v',
+		'Waze'        => 'fab fa-waze',
+		'WhatsApp'    => 'fab fa-whatsapp',
+		'Messenger'   => 'fab fa-facebook-messenger',
+		'Telephone'   => 'fas fa-phone-alt',
+		'Email'       => 'fas fa-envelope',
+	];
+
+	public static function get_mapping( string $platform ): string {
```

Comment:
maybe check if an index exists?

Reply:
@TzviRabinovitch sure, just wasn't sure how to handle the case when the index is not present., since this is a public plugin. Normally I would throw an Exception and catch it, but I have not seen that pattern in yout code, or I could return an empty string, what do you suggest?

Reply:
I think that Empty string will be ok , or maybe return false 
@louis WDYT ?

Reply:
@TzviRabinovitch implemented